### ISSUE TITLE
css doesn't play nicely with dark firefox themes

### ISSUE THREAD

**Issue by Boruch-Baum** [Rob--W/cookie-manager#24]
On the main page for a site, the page renders correctly white text on black background, but the boxes for cookie content are white background, meaning the text is invisible.

On the 'new' tab, all text is white on white background.

**Comment by Rob--W** [comment]
I installed https://addons.mozilla.org/en-US/firefox/addon/native-dark/ , but don't see anything out of the ordinary.

Can you describe how I can reproduce the bad appearance? Please add screenshots too.

**Comment by Rob--W** [comment]
Extensions cannot directly modify the CSS of other extensions. And themes generally apply to the browser UI only, not the content.

How exactly did you change the color of the content? Is it via `about:preferences` > General > Language and Appearance > Colors ?

**Comment by Boruch-Baum** [comment]
Well, for the screenshot that I posted there were no relevant add-ons enabled except for the one that you suggested. Otherwise, the only thing that I'm using that affects color are my GTK/QT themes. That'sfor the screenshot.

When I originally made the report, I was using an extension that changed the css colors of web pages, but like I said, I had that extension disabled when I prepared the screenshot.

In page `about:preferences` / colors, it is true that I do have the settings as white on black background, but I also have the over-ride setting to never use those colors when a website explicitly has its own colors. As I said above, the way I change content is by using one of the many extensions that do that. Here are a few:

https://addons.mozilla.org/en-US/firefox/addon/nightreader/?src=search

https://addons.mozilla.org/en-US/firefox/addon/darkreader/?src=search

https://addons.mozilla.org/en-US/firefox/addon/togglewebsitecolors/
